pylua/network.lua

```diff
@@ -0,0 +1,22 @@
+package.loadlib("libpython2.7.so", "*")
+require('lua-python')
+
+python.execute("import thread")
+python.execute("import time")
+python.execute("import lua")
+python.execute("from pylua.network import *")
+
+WebSocketClient = {}
+
+WebSocketClient.__index = WebSocketClient
+
+function WebSocketClient.WebSocketClient()
+	return python.eval("LuaWebSocketClient()")
+end
+
+function startPythonWebSocketServer()
+	print('start python server')
+	python.execute("server = LuaWebSocketServer()")
+	python.execute("thread.start_new_thread(server.RunServer, ())")
```

Comment:
@OHerasym, i think that the following steps should be encapsulated in the server socket
1) create socket
2) run socket in background thread
3) wait until socket is up-and-running and only then return